backend/src/modules/users/controllers/users/users.controller.ts

```diff
@@ -46,4 +49,30 @@ export class UsersController {
     }
   }
 
+  @UseGuards(AuthGuard)
+  @Patch()
+  async editUser(
+    @Body() userDto: EditUserDto,
+    @Request() request: express.Request,
+    @Response() response: express.Response,
+  ) {
+    const userId: string = request.signedCookies[userIdCookieKey];
+    userDto.id = userId;
+    const validateResult = await this.usersService.validate(userDto);
+    if (validateResult.errors) {
+      response.status(400);
+      response.send({
+        success: false,
+        status: 'invalid_form',
```

Comment:
too general name for error. Should be unique, short, very expressive.
Maybe 'incorrect_user_edit_data' or smth like it